pyproject.toml

```diff
@@ -1,3 +1,19 @@
 [build-system]
-requires = ["setuptools>=40.6.0"]
+requires = ["setuptools >= 44", "wheel >= 0.30", "setuptools_scm[toml]>=5"]
 build-backend = "setuptools.build_meta"
+
+[tool.black]
+line-length = 120
+
+[tool.isort]
+profile = "black"
+known_first_party = ["platformdirs"]
+
+[tool.setuptools_scm]
+write_to = "src/platformdirs/version.py"
+write_to_template = """
+\"\"\" Version information \"\"\"
+
+__version__ = "{version}"
```

Comment:
Personally given we have `import-metadata` which works well, I actually think it's better not to write this out these days.

Reply:
I kinda like the explicit aspect of it and makes it backwards compatible :thinking:

Reply:
fair enough, doesn't matter too much so happy if you leave it (it just means we ourselves get to maintain it obviously)